### ISSUE TITLE
'Editor' object has no attribute 'insertMathajaxChemistry' error on opening browse window

### ISSUE THREAD

**Issue by paanvaannd** [Liresol/anki-custom-shortcuts#35]
I disabled all add-ons and re-enabled them one-by-one and reloaded the Anki application between changes, and this is the only one that triggers this warning.

This occurs when the add-on is enabled in conjunction with other add-ons and while it is the only add-on enabled, which leads me to believe it is not an add-on conflict issue. To reproduce the issue, I need only enable this add-on; fixing the issue is achieved by disabling the add-on.

The shortcuts are unchanged from default values, as the error came about before I made any changes.

I have included the debug info presented by Anki on opening the browse window below:

```
Debug info:
Anki 2.1.0beta36 Python 3.6.7 Qt 5.9.5 PyQt 5.10.1
Platform: Linux
Flags: frz=False ao=True        

Caught exception:
  File "/usr/share/anki/aqt/webview.py", line 27, in cmd
    self.onCmd(str)
  File "/usr/share/anki/aqt/webview.py", line 71, in _onCmd
    self._onBridgeCmd(str)
  File "/usr/share/anki/aqt/webview.py", line 306, in _onBridgeCmd
    self.onBridgeCmd(cmd)
  File "/usr/share/anki/aqt/toolbar.py", line 54, in _linkHandler
    self.link_handlers[link]()
  File "/usr/share/anki/aqt/toolbar.py", line 72, in _browseLinkHandler
    self.mw.onBrowse()
  File "/usr/share/anki/aqt/main.py", line 797, in onBrowse
    aqt.dialogs.open("Browser", self)
  File "/usr/share/anki/aqt/__init__.py", line 78, in open
    instance = creator(*args)
  File "<decorator-gen-20>", line 2, in __init__
  File "/usr/share/anki/anki/hooks.py", line 66, in decorator_wrapper
    return repl(*args, **kwargs)
  File "/usr/share/anki/anki/hooks.py", line 57, in repl
    old(*args, **kwargs)
  File "<decorator-gen-12>", line 2, in __init__
  File "/usr/share/anki/anki/hooks.py", line 66, in decorator_wrapper
    return repl(*args, **kwargs)
  File "/usr/share/anki/anki/hooks.py", line 57, in repl
    old(*args, **kwargs)
  File "/usr/share/anki/aqt/browser.py", line 390, in __init__
    self.setupEditor()
  File "/usr/share/anki/aqt/browser.py", line 616, in setupEditor
    self.mw, self.form.fieldsArea, self)
  File "<decorator-gen-34>", line 2, in __init__
  File "/usr/share/anki/anki/hooks.py", line 66, in decorator_wrapper
    return repl(*args, **kwargs)
  File "/usr/share/anki/anki/hooks.py", line 57, in repl
    old(*args, **kwargs)
  File "/usr/share/anki/aqt/editor.py", line 55, in __init__
    self.setupShortcuts()
  File "/home/{USER}/.local/share/Anki2/addons21/24411424/custom_shortcuts.py", line 167, in cs_editor_setupShortcuts
    (config_scuts["editor insert mathjax chemistry"], self.insertMathjaxChemistry),
<class 'AttributeError'>: 'Editor' object has no attribute 'insertMathjaxChemistry'
```

**Comment by paanvaannd** [comment]
Is this related to #26?

**Comment by Liresol** [comment]
I think this is because of the Anki version, as the error is complaining that shortcut being modified isn't present in Anki 2.1.0beta36. This can probably be fixed by updating Anki to 2.1.1+, but I can try to add backwards compatibility if that's preferred.

**Comment by paanvaannd** [comment]
That may be the issue! I would much appreciate and would not mind waiting for a backport, if possible. This is the latest version that is included with the current Ubuntu LTS (18.04), so others may have this issue if they are relying on the package manager's version.

I am aware that the Anki website recommends the package downloaded from the website rather than the native package manager version because the website version is newer. However, I like having the package managed by `apt`. All of my other add-ons work properly with it, and the apt version also detects and uses the Gtk theme properly already while the website version may not, as stated on the website itself.

**Comment by Liresol** [comment]
Just pushed out a new update and it fixes the problem on version 2.1.0rc2 (which produced the same error). Hopefully this should fix things on your end as well.

**Comment by paanvaannd** [comment]
That did the trick; thank you so much for your speedy work on this!
